setup.py

```diff
@@ -1,10 +1,12 @@
+from subprocess import call
 from setuptools import find_packages, setup
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
 extras_require = {
     'update_checker': ['alteryx-open-src-update-checker >= 2.0.0'],
+    'prophet': ['cmdstan-builder == 0.0.3']
```

Comment:
This links to the `cmdstan-builder` pip package.
The link for it on GitHub is [here](https://github.com/alteryx/cmdstan_ext), and the PyPI package is [here](https://pypi.org/project/cmdstan-builder/).
To learn more about the thought process behind this, please take a look at [this](https://alteryx.atlassian.net/wiki/spaces/PS/pages/551944193/Prophet+Installation), [this](https://alteryx.atlassian.net/wiki/spaces/PS/pages/901644585/Interaction+between+Stan+wrappers+and+Prophet), [this](https://alteryx.atlassian.net/wiki/spaces/PS/pages/884441406/Cmdstan+Installation+Flows), [this](https://alteryx.atlassian.net/wiki/spaces/PS/pages/629899414/Alternate+repository+for+cmdstan+building+and+publishing+a+wheel+for+EvalML+to+use), [this](https://alteryx.atlassian.net/wiki/spaces/PS/pages/561086672/Pip+installation+of+cmdstan) and [this](https://alteryx.quip.com/aBgLAxP5WoiE/Licensing-Prophet).